arangod/Graph/NeighborsEnumerator.h

```diff
@@ -33,11 +34,11 @@ namespace graph {
 // @brief Enumerator optimized for neighbors. Does not allow edge access
 
 class NeighborsEnumerator final : public arangodb::traverser::PathEnumerator {
-  std::unordered_set<arangodb::velocypack::StringRef> _allFound;
-  std::unordered_set<arangodb::velocypack::StringRef> _currentDepth;
-  std::unordered_set<arangodb::velocypack::StringRef> _lastDepth;
-  std::unordered_set<arangodb::velocypack::StringRef>::iterator _iterator;
-  std::unordered_set<arangodb::velocypack::StringRef> _toPrune;
+  containers::FlatHashSet<arangodb::velocypack::StringRef> _allFound;
+  containers::FlatHashSet<arangodb::velocypack::StringRef> _currentDepth;
+  containers::FlatHashSet<arangodb::velocypack::StringRef> _lastDepth;
+  containers::FlatHashSet<arangodb::velocypack::StringRef>::iterator _iterator;
```

Comment:
This iterator points into `_currentDepth`, so changing the set's pointer stability could also affect its behavior.
But in the accompanying .cpp file it seems that `_iterator` is only set after modifications have been done to `_currentDepth`. So it looks safe.